### ISSUE TITLE
How to uninstall Bear?

### ISSUE THREAD

**Issue by fish830617** [rizsotto/Bear#151]
Hello, I just built Bear on my Ubuntu 16.04, now I want to remove it but I don't know how to do.
I build Bear following your instruction:

cmake $BEAR_SOURCE_DIR
make all
make install # to install
make check   # to run tests
make package # to make packages

**Comment by rizsotto** [comment]
Please read Ubuntu docs how to remove package. (It's also not clear from your description that you were installed the package or not.) Sorry.

**Comment by fish830617** [comment]
Sorry for replying a closed issue.
I have just solved my problem; I've successfully uninstall Bear on my Ubuntu!
If I would have installed Bear through `apt-get install`, I could remove it via `apt-get purge` easily;
however, I built it directly from the source.
To remove it, I use *checkinstall*. It is a `make install` equivalent and also create a Debian package with those files, so I can remove it easily through `dpkg -r bear`!

Below is the steps:
1. Install *checkinstall*.
```
sudo apt-get install checkinstall
```
2. In the Bear source directory, using `checkinstall` as `make install`
```
cd $BEAR_SOURCE_DIRECTORY
sudo checkinstall
```
3. After creating a .deb file successfully, removing Bear just like other packages!
```
sudo dpkg -r bear
```

For more information about *checkinstall*, see
https://help.ubuntu.com/community/CheckInstall

Sorry for the unclarity of my original post.
And sorry to disturb you again.
